C/CyrusSASL/build_tarballs.jl

```diff
@@ -0,0 +1,64 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "CyrusSASL"
+version = v"2.1.27"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/cyrusimap/cyrus-sasl/releases/download/cyrus-sasl-2.1.27/cyrus-sasl-2.1.27.tar.gz", "26866b1549b00ffd020f188a43c258017fa1c382b3ddadd8201536f72efb05d5"),
+    DirectorySource("./bundled"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/cyrus-sasl-*/
+if [[ "${target}" == *-mingw* ]]; then
+  atomic_patch -p1 ../patches/02-exeext.patch
+  atomic_patch -p1 ../patches/03-fix-plugins.patch
+  atomic_patch -p1 ../patches/04-manpage-paths.patch
+  atomic_patch -p1 ../patches/14-MinGW-w64-add-LIBSASL_API-to-function-definitions.patch
+  atomic_patch -p1 ../patches/15-MinGW-w64-define-LIBSASL_EXPORTS_eq_1-for-sasldb.patch
+  atomic_patch -p1 ../patches/16-MinGW-w64-define-WIN32_LEAN_AND_MEAN-avoiding-handle_t-redef.patch
+  atomic_patch -p1 ../patches/17-MinGW-w64-define-S_IRUSR-and-S_IWUSR.patch
+  atomic_patch -p1 ../patches/19-paths-relocation.patch
+  atomic_patch -p1 ../patches/20-mingw-tchar.patch
+  atomic_patch -p1 ../patches/21-fix-getopt-guard.patch
+
+  cp win32/include/md5global.h include/md5global.h
```

Comment:
It looks like it's using the original `include/md5global.h` rather than the one that you're copying

Reply:
Yeah, this file is deleted during [`make`](https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=5878&view=logs&j=e7e877bf-8e27-5cc6-37ef-ebbcef74f0da&t=2de339ae-1031-5827-a669-8653b6053df6&l=1709):
```
[10:28:50] rm -f md5global.h
[10:28:50] ./makemd5 md5global.h
```